ci-scripts/update-ccv2.sh

```diff
@@ -0,0 +1,177 @@
+mkdir -p "$CCV2_B2C_STOREFRONT_PATH/dist/$B2C_STORE/server"
+
+# b2b
+mkdir -p "$CCV2_B2B_STOREFRONT_PATH/dist/$B2B_STORE/browser"
+mkdir -p "$CCV2_B2B_STOREFRONT_PATH/dist/$B2B_STORE/server"
+
+echo "------------------------------------------------------------------"
+echo "Build Spartacus libraries"
+yarn build:libs
+
+echo "------------------------------------------------------------------"
+echo "update server.ts for B2C storefront"
+
+sed -i "s%dist/storefrontapp%dist/$B2C_STORE/browser%gi" "$SERVER_CONFIG_PATH"
+
+echo "------------------------------------------------------------------"
+echo "Verify server.ts has been updated for B2C dist"
+
+cat "$SERVER_CONFIG_PATH"
+
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2C_STORE/browser');" "$SERVER_CONFIG_PATH"
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "------------------------------------------------------------------"
+echo "Build SSR for B2C storefront"
+
+yarn build:ssr:ci
+
+echo "------------------------------------------------------------------"
+echo "Build CSR for B2C storefront"
+
+yarn build
+
+echo "------------------------------------------------------------------"
+echo "Copy server and browser files to js-storefront to adhere to the ccv2 dist structure for B2C storefront"
+
+copy_browser_and_server_files "$CCV2_B2C_STOREFRONT_PATH $B2C_STORE"
+
+echo "------------------------------------------------------------------"
+echo "update server.ts for B2B storefront"
+
+sed -i "s%dist/$B2C_STORE/browser%dist/$B2B_STORE/browser%gi" "$SERVER_CONFIG_PATH"
+
+echo "------------------------------------------------------------------"
+echo "Verify server.ts has been updated for B2B dist"
+
+cat "$SERVER_CONFIG_PATH"
+
+if grep -Fq "const distFolder = join(process.cwd(), 'dist/$B2B_STORE/browser');" "$SERVER_CONFIG_PATH"
+then
+    echo "Dist folder has been updated"
+else
+    echo "Dist folder has NOT been updated"
+    exit 1
+fi
+
+echo "------------------------------------------------------------------"
+echo "Build SSR for B2B storefront"
+
+export SPA_ENV='b2b'
+yarn build:ssr:ci
+
+echo "------------------------------------------------------------------"
+echo "Build CSR for B2B storefront"
+
+yarn build
+
+echo "------------------------------------------------------------------"
+echo "Copy server and browser files to js-storefront to adhere to the ccv2 dist structure for B2B storefront"
+
+copy_browser_and_server_files "$CCV2_B2B_STOREFRONT_PATH" "$B2B_STORE"
+
+echo "------------------------------------------------------------------"
+echo "Push to remote repository"
+
+cd "$GHT_REPO"
+
+git config --global user.email dl_61c08cf5a3cac30261c7e88c@global.corp.sap
+git config --global user.name cx-cc-automation-serviceuser
+
+git add .
+git commit --allow-empty -m "Update with $SOURCE_BRANCH_TO_DEPLOY branch from source of Spartacus" 
```

Comment:
Not really an issue, but it doesn't seem like we need `--allow-empty` here.

Reply:
I believe there were times when there's nothing to update, for example, redeploying a branch twice when there was no change, so it would fail the action saying there is nothing to push.

Me putting the allow-empty is to just know that someone triggered it if ever it's empty, and to make the action green instead of red

Reply:
Agreed.  If it is there to add robustness, then we keep it for sure..